images/bootstrap/Dockerfile

```diff
@@ -14,6 +14,37 @@
 # limitations under the License.
 
 # Includes basic workspace setup, with gcloud and a bootstrap runner
+FROM fedora:36 AS builder
+
+ENV PKG_CONFIG_PATH="/usr/lib/pkgconfig"
+ENV PODMAN_VERSION="v4.0.3"
```

Comment:
can we use please older version ? for example 3.4 ?
I have this error https://github.com/containers/podman/issues/13327 on CI [1]
and one mentioned WA is to downgrade

[1] https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2125/rehearsal-pull-cluster-network-addons-operator-unit-test/1529420827697614848

Locally i am using podman 3.4.4 and all works good
Thanks

Reply:
This part of the Dockerfile is used to build podman v4.0.3 

This may no longer needed as there is newer versions of podman available in the fedora repo that wouldn't have been available when the original PR was made. 

I think I will try to run a couple of tests with the version in the fedora 36 repos and we may be able to remove this builder section altogether.

Its podman v4.1.0 in Fedora 36 at the moment  - if that doesn't work we can look at building an older release